src/test/java/reactor/netty/http/server/HttpServerTests.java

```diff
@@ -554,38 +558,95 @@ public void testContentLengthHeadRequest() {
-		doTestContentLengthHeadRequest("/1", disposableServer.address(), HttpMethod.HEAD, true, false);
-		doTestContentLengthHeadRequest("/2", disposableServer.address(), HttpMethod.GET, false, true);
-		doTestContentLengthHeadRequest("/2", disposableServer.address(), HttpMethod.HEAD, false, true);
-		doTestContentLengthHeadRequest("/3", disposableServer.address(), HttpMethod.GET, false, false);
-		doTestContentLengthHeadRequest("/3", disposableServer.address(), HttpMethod.HEAD, false, false);
-		doTestContentLengthHeadRequest("/4", disposableServer.address(), HttpMethod.HEAD, true, false);
-		doTestContentLengthHeadRequest("/5", disposableServer.address(), HttpMethod.HEAD, false, true);
-		doTestContentLengthHeadRequest("/6", disposableServer.address(), HttpMethod.HEAD, false, false);
-		doTestContentLengthHeadRequest("/7", disposableServer.address(), HttpMethod.HEAD, true, false);
-		doTestContentLengthHeadRequest("/8", disposableServer.address(), HttpMethod.HEAD, false, true);
-		doTestContentLengthHeadRequest("/9", disposableServer.address(), HttpMethod.HEAD, false, false);
+		doTestContentLengthHeadRequest("/1", disposableServer.address(), HttpMethod.GET, sentHeaders, true, false);
+		doTestContentLengthHeadRequest("/1", disposableServer.address(), HttpMethod.HEAD, sentHeaders, true, false);
+		doTestContentLengthHeadRequest("/2", disposableServer.address(), HttpMethod.GET, sentHeaders, false, true);
+		doTestContentLengthHeadRequest("/2", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, true);
+		doTestContentLengthHeadRequest("/3", disposableServer.address(), HttpMethod.GET, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/3", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/4", disposableServer.address(), HttpMethod.HEAD, sentHeaders, true, false);
+		doTestContentLengthHeadRequest("/5", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, true);
+		doTestContentLengthHeadRequest("/6", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/7", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/8", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/9", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/10", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/11", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/12", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+		doTestContentLengthHeadRequest("/13", disposableServer.address(), HttpMethod.HEAD, sentHeaders, false, false);
+	}
+
+	@Test
+	public void testIssue() {
```

Comment:
that would benefit from a more descriptive name :)

Reply:
oppps :)